### DISCUSSION TITLE
[@types/webextension-polyfill] wrong return type

### DISCUSSION THREAD

**Discussion by ntnyq** [DefinitelyTyped/DefinitelyTyped#71295] [Issues with a @types package]
`browser.action.setBadgeTextColor` and `browser.action.getBadgeTextColor` 

should return `Promise<void>` rather than `void`.

[a comment by an automated account is not shown]

**Comment by hkleungai** [comment]
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71963.
